src/libcore/iter/range.rs

```diff
@@ -353,6 +304,23 @@ impl<A: Step> Iterator for ops::RangeFrom<A> where
     fn size_hint(&self) -> (usize, Option<usize>) {
         (usize::MAX, None)
     }
+
+    #[inline]
+    fn count(self) -> usize {
+        usize::MAX
```

Comment:
As the `Iterator` implementation for `RangeFrom` in fundamentally broken at the moment (#25708), I don't think it's a good idea to override `count`, `nth` and `last` just yet. At least the default impls match what happens when you repeatedly call `.next()`.